packages/domains-analytics-writer/src/config/config.ts

```diff
@@ -10,9 +10,41 @@ import {
   KafkaBatchConsumerConfig,
   LoggerConfig,
   EServiceTemplateTopicConfig,
+  AnalyticsSQLDbConfig,
+  ReadModelSQLDbConfig,
 } from "pagopa-interop-commons";
 import { z } from "zod";
 
+// We explicitly set these environment variables here because
+// the `pagopa-interop-readmodel-models` and `pagopa-interop-readmodel` packages
+// require a fully configured READMODEL_SQL_DB environment in order to work.
+// so we must be sure that env is set up before any import or usage happens.
+
+// These packages are needed to use splitter into SQL functions like `splitAttributeIntoObjectsSQL` and models like `AttributeSQL`,
+export const readModelSetupEnv = {
+  READMODEL_SQL_DB_HOST: "localhost",
+  READMODEL_SQL_DB_PORT: "6002",
+  READMODEL_SQL_DB_USERNAME: "root",
+  READMODEL_SQL_DB_PASSWORD: "root",
+  READMODEL_SQL_DB_NAME: "root",
+  READMODEL_SQL_DB_USE_SSL: "false",
+  READMODEL_SQL_DB_SCHEMA_AGREEMENT: "readmodel_agreement",
+  READMODEL_SQL_DB_SCHEMA_ATTRIBUTE: "readmodel_attribute",
+  READMODEL_SQL_DB_SCHEMA_CATALOG: "readmodel_catalog",
+  READMODEL_SQL_DB_SCHEMA_CLIENT_JWK_KEY: "readmodel_client_jwk_key",
+  READMODEL_SQL_DB_SCHEMA_CLIENT: "readmodel_client",
+  READMODEL_SQL_DB_SCHEMA_DELEGATION: "readmodel_delegation",
+  READMODEL_SQL_DB_SCHEMA_ESERVICE_TEMPLATE: "readmodel_eservice_template",
+  READMODEL_SQL_DB_SCHEMA_PRODUCER_JWK_KEY: "readmodel_producer_jwk_key",
+  READMODEL_SQL_DB_SCHEMA_PRODUCER_KEYCHAIN: "readmodel_producer_keychain",
+  READMODEL_SQL_DB_SCHEMA_PURPOSE: "readmodel_purpose",
+  READMODEL_SQL_DB_SCHEMA_TENANT: "readmodel_tenant",
+};
```

Comment:
Is this because Drizzle requires these vars to be set to generate the models?

@taglioni-r, if yes, why not hardcode these directly in those packages (readmodel-models and readmodel)?

We could have, in those packages, two different sets of variables: 1) the ones used for model generation, required at build time, hardcoded inside `readmodel` and `readmodel-models`, pointing to the local DB, and 2) these ones actually used to connect to the DB